README.md

```diff
@@ -1,5 +1,10 @@
 # @vexchange/governance
 
 
+## Deployment
+
+Place `.env` with the private key in the root directory under the `variable PRIVATE_KEY=0x000...abc`
```

Comment:
Should it be variable `PRIVATE_KEY=0x000...abc` instead of `variable PRIVATE_KEY=0x000...abc`

Reply:
yes, thanks for catching this